pkg/convert/info.go

```diff
@@ -75,22 +75,24 @@ func (s *mapperProviderInfoSource) GetProviderInfo(
 	if isTerraformProvider(pulumiProvider) && requiredProvider != nil {
 		tfVersion, diags := shim.FindTfPackageVersion(requiredProvider)
 		if diags.HasErrors() {
-			return nil, fmt.Errorf("could not find version for terraform provider %s: %v", tfProvider, diags)
```

Comment:
We should probably add a warning diagnostic here, but I think it's fine to still try and do the conversion without a hint.

Reply:
I'm happy to pass a warning, but what do you suggest to use for "without a hint"? Perhaps I'm missing something about how hints are used but I was going for "most obvious default".

Reply:
Something like:
"Could not find TF version for '<requiredProvider>' assuming a Pulumi provider '<pulumiProvider>' exists."

Reply:
ohhhh you just want to add a log line explaining we're going with the assumption proposed. Gotcha, thank you!